.gitconfig

```diff
@@ -8,28 +8,53 @@
 	email = caziz@ucla.edu
 [core]
 	excludesfile = /Users/christopheraziz/.gitignore_global
+	# I know I said something in class, but. Loooool xD I respect this.
 	editor = subl -n -w
 
+# I changed these spaces to tabs. It doesn't matter, but I did it anyway. Tbh I
+# don't know what I'm making these chagnes for, I'm only submitting this pull
+# request so you can see m y commends on your code in an organized fashion, but
+# I guess you can merge some hunks if you want? Lol idk how to merge only
+# specific hunks. But anyway, I'm all about advanced usage of git (and GitHub)
+# - I wonder what a GUI-less pull request is supposed to look like?? Anyway,
+# lit colors :')
```

Comment:
I found recently that I like using tabs but having the IDE insert spaces but that might be just a fad